protocol/Cargo.toml

```diff
@@ -21,6 +21,9 @@ futures = { version = "0.3.30", default-features = false, optional = true }
 tokio = { version = ">=1.37.0, <1.39.0", default-features = false, optional = true }
 rand = { version = "0.8.0", default-features = false }
 bitcoin = { version = "0.32.4", default-features = false }
+# Depending on hashes directly for HKDF, can drop this and 
+# use the re-exported version in bitcoin > 0.32.*.
+bitcoin_hashes = { version ="0.15.0" }
```

Comment:
`bitcoin_hashes = { version = "0.15.0", default-features = false }` should do the trick